From 732bbb53d829bf7ceaf86995b8e7d8603003aaad Mon Sep 17 00:00:00 2001
From: Vineeth Upadhya K <vineeth.k.upadhya@intel.com>
Date: Mon, 8 Oct 2012 12:10:34 +0530
Subject: [PORT FROM MAIN] wifi: Revert "wifi: increase ps_poll_timeouts"

BZ: 43286

This partially reverts commit 71f8735af99905f4b419f30f4e520a24305c3b77

Change-Id: Icd5234253dab72d7e8aea116d03a727477361cd5
Orig-Change-Id: I894b7a9fe673dd46ec12ee16817cb81985d745a5
Signed-off-by: Frode Isaksen <frodex.isaksen@intel.com>
               Vineeth Upadhya K <vineeth.k.upadhya@intel.com>
Signed-off-by: Jonathan DE CESCO <jonathan.de.cesco@intel.com>
Reviewed-on: http://android.intel.com:8080/69209
Reviewed-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: Upadhya, Vineeth K <vineeth.k.upadhya@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
Reviewed-on: http://android.intel.com:8080/77992
Reviewed-by: Li, XiaojingX <xiaojingx.li@intel.com>
Tested-by: Li, XiaojingX <xiaojingx.li@intel.com>
---
 wl12xx-compat/drivers/net/wireless/wl12xx/main.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/wl12xx-compat/drivers/net/wireless/wl12xx/main.c b/wl12xx-compat/drivers/net/wireless/wl12xx/main.c
index c7ac16a..94af846 100644
--- a/wl12xx-compat/drivers/net/wireless/wl12xx/main.c
+++ b/wl12xx-compat/drivers/net/wireless/wl12xx/main.c
@@ -125,8 +125,8 @@ static struct conf_drv_settings default_conf = {
 	.rx = {
 		.rx_msdu_life_time           = 512000,
 		.packet_detection_threshold  = 0,
-		.ps_poll_timeout             = 80,
-		.upsd_timeout                = 80,
+		.ps_poll_timeout             = 15,
+		.upsd_timeout                = 15,
 		.rts_threshold               = IEEE80211_MAX_RTS_THRESHOLD,
 		.rx_cca_threshold            = 0,
 		.irq_blk_threshold           = 0xFFFF,
-- 
1.7.4.1

